### PR TITLE
fix: [CDS-46281]: Jenkins service step log fix

### PR DESCRIPTION
**Pull request by vivekkarnatiharness**
## Describe your changes
This is fix for improving log message for service step,when we use jenkins as primary artifact source
## Checklist
- [ ] I've documented the changes in the PR description.
- [ ] I've tested this change either in PR or local environment.
- [ ] If hashcheck failed I followed [the checklist](https://harness.atlassian.net/wiki/spaces/DEL/pages/21016838831/PR+Codebasehash+Check+merge+checklist) and updated this PR description with my answers to make sure I'm not introducing any breaking changes.

## Comment Triggers
<details>
  <summary>Build triggers</summary>
  
- Feature build: `trigger feature-build`
- Immutable delegate `trigger publish-delegate`
</details>

<details>
  <summary>PR Check triggers</summary>
  
  You can run multiple PR check triggers by comma separating them in a single comment. e.g. `trigger ti0, ti1`
  
- Compile: `trigger compile`
- CodeFormat: `trigger codeformat`
- MessageMetadata: `trigger messagecheck`
- Recency: `trigger recency`
- BuildNumberMetadata: `trigger buildnum`
- runDockerizationCheck: `trigger dockerizationcheck`
- runAuthorCheck: `trigger authorcheck`
- Checkstyle: `trigger checkstyle`
- PMD: `trigger pmd`
- Feature Name Check: `trigger featurenamecheck`
- TI-bootstrap: `trigger ti0`
- TI-bootstrap1: `trigger ti1`
- TI-bootstrap2: `trigger ti2`
- TI-bootstrap3: `trigger ti3`
- TI-bootstrap4: `trigger ti4`
- FunctionalTest1: `trigger ft1`
- FunctionalTest2: `trigger ft2`
- CodeBaseHash: `trigger codebasehash`
- Trigger all Checks: `trigger smartchecks`
</details>

## PR check failures and solutions
https://harness.atlassian.net/wiki/spaces/BT/pages/21106884744/PR+Checks+-+Failures+and+Solutions

## [Contributor license agreement](https://github.com/harness/harness-core/blob/develop/CONTRIBUTOR_LICENSE_AGREEMENT.md)

- - -
This change is [Reviewable](https://reviewable.io/reviews/harness/harness-core/40110)

### CODE REVIEW

**Comment by vivekkarnatiharness** [comment]
trigger messagecheck

**Comment by vivekkarnatiharness** [comment]
trigger messagecheck

**Comment by vivekkarnatiharness** [comment]
trigger messagecheck

**Comment by vivekkarnatiharness** [comment]
trigger codeformat

File: 127-cd-nextgen-entities/src/main/java/io/harness/cdng/artifact/utils/ArtifactUtils.java [hunk 1]
```diff
@@ -178,9 +178,9 @@ public String getLogInfo(ArtifactConfig artifactConfig, ArtifactSourceType sourc
         return String.format("type: %s, build: %s", sourceType, customArtifactConfig.getVersion().getValue());
       case JENKINS:
         JenkinsArtifactConfig jenkinsArtifactConfig = (JenkinsArtifactConfig) artifactConfig;
-        return String.format(placeholder, sourceType, jenkinsArtifactConfig.getJobName().getValue(),
-            jenkinsArtifactConfig.getArtifactPath().getValue(), jenkinsArtifactConfig.getBuild().getValue(),
-            jenkinsArtifactConfig.getConnectorRef().getValue());
+        return String.format("\ntype: %s \nJobName: %s \nArtifactPath: %s \nBuild: %s \nConnectorRef: %s\n", sourceType,
```
Comment (puthrayaharness) [inline 127-cd-nextgen-entities/src/main/java/io/harness/cdng/artifact/utils/ArtifactUtils.java:181]: What is source type here?

Comment (vivekkarnatiharness) [reply 127-cd-nextgen-entities/src/main/java/io/harness/cdng/artifact/utils/ArtifactUtils.java:181]: Source type can be JENKINS or ACR ..

**Review by shivamnegi94 — APPROVED** [review]

**Review by shivamnegi94 — APPROVED** [review]
> Reviewed 2 of 2 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/40110)* status: 1. unresolved discussions.

**Comment by vivekkarnatiharness** [comment]
*[`127-cd-nextgen-entities/src/main/java/io/harness/cdng/artifact/utils/ArtifactUtils.java` line 181 at r1](https://reviewable.io/reviews/harness/harness-core/40110#-NGqQYPaSWAW8qGD-K4L-r1-181:-NGtgQGC4w3CX9D3QoVO:b-896fix) ([raw file](https://github.com/harness/harness-core/blob/17799da1e4aea32de432bd14830992a8483e7c2e/127-cd-nextgen-entities/src/main/java/io/harness/cdng/artifact/utils/ArtifactUtils.java#L181)):*
<details><summary><i>Previously, vivekkarnatiharness wrote…</i></summary><blockquote>

Source type can be JENKINS or ACR ..
</blockquote></details>

Done.

**Review by puthrayaharness — APPROVED** [review]

**Review by puthrayaharness — APPROVED** [review]
> Reviewed 2 of 2 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/40110)* status: :shipit: complete! ✓

**Pull request merged**